index.html

```diff
@@ -11,9 +11,148 @@
+          <p class="program-item-title">Keynote Speech</p>
+          <p class="list-para">Listen to renowed Psychologists talk about personal experiences.</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-book-open program-icon"></i>
+          <p class="program-item-title">Reseach papers</p>
+          <p class="list-para">Submit your Research paper for free professional review by the present professionals.</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-walking program-icon"></i>
+          <p class="program-item-title">Walking away</p>
+          <p class="list-para">Control is part of what makes and destroys you, we can teach you to contol control</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-user-friends program-icon"></i>
+          <p class="program-item-title">Seminar</p>
+          <p class="list-para">Partner organized skills building workshops</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item" id="join">
+          <i class="far fa-flag program-icon"></i>
+          <p class="program-item-title">Nigeria and Mental Health</p>
+          <p class="list-para">The race and fight will only get better when fought as one</p>
+        </li>
+      </ul>
+      <!-- REGISTRATION BUTTON -->
+      <div class="d-flex justify-center">
+        <button class="button register-button" type="button">Join Psych Conference</button>
+      </div>
+    </section>
+      <!-- FEATURED SPEAKERS -->
+    <section>
+      <div class="container">
+        <div class="speakers">
+          <h1 class="speakers-heading">Featured Speakers</h1>
+          <ul class="speakers-list" id="speakers-list">
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="partners text-center">
+      <h1 class="pt-4 partners-heading">Partner</h1>
+      <ul class="partners-list d-flex flex-row justify-content-between m-5">
+        <li>
+          <img class="img-fluid" src="images/google.jpg" alt="Google Partner">
+        </li>
+        <li>
+          <img class="img-fluid" src="images/airbnb.png" alt="AirBnB Partner">
+        </li>
+        <li>
+          <img class="img-fluid" src="images/amazon.png" alt="Amazon Partner">
+        </li>
+        <li>
+          <img class="img-fluid" src="images/facebook.png" alt="Facebook Parner">
+        </li>
+      </ul>
+    </section>
+  </main>
+   <footer class="d-flex gap-4 fs-6 p-3">
+     <div class="footer-logo">
+      <img src="images/psychology-logo.jpg" alt="Psychology Logo" width="50" height="50" class="d-inline-block align-text-top">
+    </div>
+    <div class="rights">
+      <p class="fw-bold text-center">2021 Psych Conference | Health <br /> Some Rights Reserved</p>
+    </div>
+   </footer>
```

Comment:
- [ ] **OPTIONAL**
Currently, the footer's elements' alignment doesn't match the original design. It'd be better if you aligned the elements as the original design 👍